### PR TITLE
Add little-cheesemonger logo

### PR DESCRIPTION
**Pull request by rayz**
This PR adds the little-cheesemonger logo to the README.md file. I have placed the `logo.svg` file in the root instead of an `assets` folder because we only have one image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
 correctly linked manylinux wheel. Configuration data is loaded from a `pyproject.toml` file, or 
 a custom data loader can be implemented.
 
+
+<p align="center">
+  <img width="300" height="300" src="logo.svg">
+</p>
```
Comment (fhightower) [inline README.md:10]: Looks good! I recommend (but you  certainly don't have to) putting the logo above the project summary (either above the title or below the title is fine) as I think it looks a little nicer (you can see some examples [here](https://github.com/gruns/icecream) and [here](https://github.com/MISP/MISP)).

**Review by fhightower — APPROVED** [review]
Nice logo!

File: README.md (hunk 1 above)
Comment (rayz) [reply README.md]: Done! Originally I was looking at the [cookiecutter repo](https://github.com/cookiecutter/cookiecutter) which had it below the summary 😅

Comment (fhightower) [reply README.md]: No worries, that works too. Personally, I think it looks better above the summary, but feel free to put it where you think is best (it's not critical either way).

Comment (rayz) [reply README.md]: Can you merge it for me? I do not have merge access. #54 needs to be merged as well. Thanks

**Pull request merged**